modular_skyrat/modules/xenoarch/code/modules/research/xenoarch/xenoarch_item.dm

```diff
@@ -2,19 +2,117 @@
+			look around, we could find some of those old jewels."
+		if(ANCIENT_COIL)
+			new_item.name = "ancient coil"
+			new_item.desc = "This useless relic is an ancient coil that dates from around [rand(400,600)] years ago. \
+			It is made of iron and copper. It has some burn marks around the iron rod. Perhaps later on, we could \
+			use it for some machines."
+		if(ANCIENT_LIGHT)
+			new_item.name = "ancient light"
+			new_item.desc = "This useless relic is an ancient light that dates from around [rand(400,600)] years ago. \
+			It is made of iron and has glass shards around it. It has dents on the iron and clear damage from misuse. \
+			Perhaps we could research this later on to see how the ancients made lights."
+		if(ANCIENT_CUP)
+			new_item.name = "ancient cup"
+			new_item.desc = "This useless relic is an ancient cup that dates from around [rand(900,1100)] years ago. \
+			It is made of hardened stone. There are small cracks all along the surface, as long as chisel marks. \
+			Perhaps it will give insight into the ancient's eating and drinking habits."
+		if(ANCIENT_UTENSILS)
+			new_item.name = "ancient utensils"
+			new_item.desc = "These useless relics are ancient utensils that dates from around [rand(900,1100)] years ago. \
+			It is made of hardened stone. There are small cracks all along the surface, as long as chisel marks. \
+			Perhaps it will give insight into the ancient's eating and drinking habits."
+		if(ANCIENT_R_BOWL)
+			new_item.name = "ancient rock bowl"
+			new_item.desc = "This useless relic is an ancient rock bowl that dates from around [rand(900,1100)] years ago. \
+			It is made of hardened stone. There are small cracks all along the surface, as long as chisel marks. \
+			Perhaps it will give insight into the ancient's eating and drinking habits."
+	qdel(src)
+
+#undef ANCIENT_URN
+#undef ANCIENT_BOWL
+#undef ANCIENT_CROWN
+#undef ANCIENT_COIL
+#undef ANCIENT_LIGHT
+#undef ANCIENT_CUP
+#undef ANCIENT_UTENSILS
+#undef ANCIENT_R_BOWL
+
+/obj/item/xenoarch/useless_relic/magnified
+	name = "magnified useless relic"
+	desc = "A useless relic that can be redeemed for cargo or research points. Has been magnified."
 
 /datum/export/xenoarch/useless_relic
-	cost = CARGO_CRATE_VALUE*2
-	unit_name = "xenoarch item"
+	cost = CARGO_CRATE_VALUE * 3 //600
+	unit_name = "useless relic"
 	export_types = list(/obj/item/xenoarch/useless_relic)
+	include_subtypes = FALSE
 
 /datum/export/xenoarch/useless_relic/sell_object(obj/O, datum/export_report/report, dry_run, apply_elastic = FALSE) //I really dont want them to feel gimped
 	. = ..()
 
+/datum/export/xenoarch/useless_relic/magnified
+	cost = CARGO_CRATE_VALUE * 6 //1200
+	unit_name = "magnified useless relic"
+	export_types = list(/obj/item/xenoarch/useless_relic/magnified)
+	include_subtypes = FALSE
+
+/datum/export/xenoarch/useless_relic/magnified/sell_object(obj/O, datum/export_report/report, dry_run, apply_elastic = FALSE) //I really dont want them to feel gimped
+	. = ..()
```

Comment:
Isn't this already handled by the parent?

Reply:
addressed